packages/ckeditor5-link/docs/_snippets/features/autolink.html

```diff
@@ -1,4 +1,5 @@
 <div id="snippet-autolink">
-	<p>Type space, or Enter or Shift+Enter after a link: https://ckeditor.com</p>
-	<p>Will link to E-mail addresses also: example@example.com</p>
-</div>
+	<p>Type <kbd>Space</kbd> after a link to see the magic of autolink: https://ckeditor.com</p>
```

Comment:
\<p>Press Space after a link to see the magic of autolinking: https://ckeditor.com\</p>